Formula/dosfstools.rb

```diff
@@ -0,0 +1,39 @@
+class Dosfstools < Formula
+  desc "Tools to create, check and label file systems of the FAT family"
+  homepage "https://github.com/dosfstools"
+
+  stable do
+    url "https://github.com/dosfstools/dosfstools/releases/download/v4.0/dosfstools-4.0.tar.gz"
+    mirror "https://fossies.org/linux/misc/dosfstools-4.0.tar.gz"
+    sha256 "25809c30403c9a945ae34827ec75df32ff55017415a720864fefccc8c8f9991f"
+
+    patch do
+      url "https://github.com/dosfstools/dosfstools/commit/1e76e5778a1885452939a79d9145b80634a5b023.patch"
+      sha256 "824c800a589db8a779fddbbd4b8eef5a2941f838ef8f46ad390a680c8cabd2a4"
+    end
+  end
+
+  head do
+    url "https://github.com/dosfstools/dosfstools.git"
+  end
+
+  option "with-symlinks", "Symlink older names of the tools to the current ones on installation"
+
+  depends_on "automake" => :build
+  depends_on "autoconf" => :build
+  depends_on "pkg-config" => :build
+
+  def install
+    system "autoreconf", "-i", "-f"
+    configure_args = ["--without-udev", "--prefix=#{prefix}"]
+    if build.with? "symlinks"
+      configure_args << "--enable-compat-symlinks"
+    end
+    system "./configure", *configure_args
+    system "make", "install"
+  end
+
+  test do
+    system "#{sbin}/mkfs.fat", "--help"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!